Update terminologies in the updated v6 docs - Closes 1658 (#1675)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/understand-blockchain/index.adoc

:url_scalability_introduction: intro/blockchain-scalability.adoc
:url_understand_interoperability: understand-blockchain/interoperability/index.adoc
:url_relayer: understand-blockchain/interoperability/index.adoc#sidechain-to-mainchain
:url_sidechain_glossary: glossary.adoc#sidechain-application

> :url_sapp: glossary.adoc#sapp

Reviewer: Pls add as footnotes

All other connected blockchains are called *sidechains* in this context.
To become interoperable with any other blockchain in the Lisk ecosystem, a blockchain needs to follow the xref:{url_understand_interoperability}[] protocol.

Each sidechain is registered to the mainchain and is specific to a sidechain application.
Every blockchain is maintained by a network of blockchain nodes.
These applications can have customized logic, UIs, and a sidechain as their application's database.

> For more information about Lisk Protocol, see {url_protocol}[Lisk Improvement Proposals].

Reviewer: After reading it several times, i am not sure what this paragraph wants to tell me. Maybe it can be removed?

== What are sidechain applications?

> A sidechain application refers to a software that contains a lisk based sidechain client, a middleware and a UI.

Reviewer: Are we explaining what a sidechain client is as well? Would explain it before or add a link to explanation here.

> More specifically, within the Lisk documentation, whenever xref:{url_sapp}[sApps] are discussed, this is directly referring to applications that run by using sidechain, which are compatible with Lisk Protocol. 

Reviewer: WIth the new terminology introduced, I think we can remove this sentence.

If the received block is verified, then the receiving node adds it to its blockchain instance.

===== Data Stores

> Each blockchain node keeps data on various data stores inside its Lisk sidechain\'s instance. This data contains the following two types: on-chain and off-chain.

Reviewer: ```suggestion
Each blockchain node is an instance of a particular sidechain and keeps the data on various data stores inside.
This data contains the following two types: on-chain and off-chain.
```

> Each blockchain node keeps data on various data stores inside its Lisk sidechain\'s instance. This data contains the following two types: on-chain and off-chain.

Reviewer: ```suggestion
Each blockchain node is an instance of a particular sidechain and keeps the data on various data stores inside.
This data contains the following two types: on-chain and off-chain.
```
